public/js/test/mochitest/head.js

```diff
@@ -225,8 +235,13 @@ function isVisibleWithin(outerEl, innerEl) {
 const selectors = {
   callStackHeader: ".call-stack-pane ._header",
   frame: (index) => `.frames ul li:nth-child(${index})`,
-  gutter: line => `.CodeMirror-code *:nth-child(${line}) .CodeMirror-linenumber`
-}
+  gutter: line => `.CodeMirror-code *:nth-child(${line}) .CodeMirror-linenumber`,
+  pauseKey: { keyCode: "VK_F8" },
+  resumeKey: { keyCode: "VK_F8" },
+  stepOverKey: { keyCode: "VK_F10" },
+  stepInKey: { keyCode: "VK_F11" },
+  stepOutKey: { keyCode: "VK_F12", modifiers: { shiftKey: true } }
```

Comment:
Not sure these belong in `selectors`? These selectors are meant to be CSS selectors, and this just seems like a key mapping. You could pull this out into it's own object next to the `pressKey` function.